### ISSUE TITLE
Option to disable deletion of final directory when importing?

### ISSUE THREAD

**Issue by kooimens** [sampsyo/beets#1553]
When I use `beet import /home/user/music/` or `beet import /home/user/music` I can't find an option to disable the deletion of the /home/user/music directory. It's ok that beets is deleting everything underneath it, but the final directory e.g. music should remain on disk.

**Comment by sampsyo** [comment]
This is actually how beets works currently—the top-level directory is *not* removed. In fact, #130 is a request to change this behavior so that it *is* removed.

Is this behavior reproducible every time? If so, this is a bug. What does your config file look like?

**Comment by kooimens** [comment]
From readthedocs (clutter):
`The importer only removes recursively searched subdirectories—the top-level directory you specify on the command line is never deleted.`

Tried several times, my top-level directory is always removed. I'v used this config file:
http://pastebin.com/HxJvthDc

**Comment by sampsyo** [comment]
Oddly, I just tried this locally (using the default config) and was unable to reproduce this.

Can you try reducing your config to see if there's anything in it that's causing this behavior?

Is there anything useful in the verbose output from one of these test runs?

**Comment by kooimens** [comment]
@sampsyo. I've simplified my config file and did some test runs. My directory in the config file is:
/mnt/wd1/files/music.

If I run beet import /mnt/wd/files/music/test. The "test" directory is removed, but I think this is correct since the test directory is within the beets music directory.

If I run beet import /home/user/test. The "test" directory is NOT removed anymore. 

I wasn't able to reproduce what was causing the issue. Closing for now.
